### ISSUE TITLE
Models are not creating the right PII values for students dataset (metadata bug)

### ISSUE THREAD

**Issue by npatki** [sdv-dev/SDV#1149]
### Environment Details

* SDV version: 0.17.2
* Python version: 3.8
* Operating System: MacOS (Darwin)

### Error Description

If I use the `student_placements_pii` demo dataset with the provided metadata, the PII column `'address'` doesn't seem to be configured properly. The synthetic data just contains letters like `'a', 'b', 'c'` instead of fake addresses like intended. 

Real Data:
<img width="1232" alt="image" src="https://user-images.githubusercontent.com/3227804/207700325-901c558b-bf86-4bd4-9911-992fb68cbd55.png">

Synthetic Data:
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/3227804/207700387-ac914311-ce91-469d-9dd6-83fb54c9515a.png">

### Steps to reproduce

```python
from sdv.demo import load_tabular_demo
from sdv.tabular import GaussianCopula

metadata, data = load_tabular_demo('student_placements_pii', metadata=True)
model = GaussianCopula(table_metadata=metadata)
model.fit(data)
synthetic_data = model.sample(num_rows=5)
```

### Details

The issue is in the returned `metadata`. It improperly specified for column `address` -- 

Current (incorrect) specification:
```python
'address': {'type': 'id',
   'subtype': 'string',
   'pii': True,
   'pii_category': 'address'},
```

Correct specification:
```python
'address': {
  'type': 'categorical',
  'pii': True,
  'pii_category': 'address'
}
```

If I correct the metadata, it works as intended.

**Comment by npatki** [comment]
There is an additional bug in the metadata for `student_placements` AND `student_placements_pii`: The `duration` column should be `numerical` but it is specified as `categorical`

**Comment by npatki** [comment]
The PII values issue has now been resolved in the new, [**SDV 1.0 (Beta!)**](https://docs.sdv.dev/sdv/) release.
